src/metabase/events/audit_log.clj

```diff
@@ -85,23 +85,37 @@
 (derive :event/subscription-unsubscribe ::pulse-event)
 (derive :event/subscription-unsubscribe-undo ::pulse-event)
 (derive :event/alert-unsubscribe ::pulse-event)
+(derive :event/subscription-create ::pulse-event)
+(derive :event/subscription-update ::pulse-event)
+
+(defn- create-details-map [pulse name]
+  (let [channels (:channels pulse)]
```

Comment:
`:channels` is array due to alerts being able to have both slack and email on the same alert.